source/community/reactnative/src/components/containers/container-wrapper.js

```diff
@@ -124,6 +140,5 @@ export class ContainerWrapper extends React.PureComponent {
 const styles = StyleSheet.create({
     backgroundImage: {
```

Comment:
Background Image is not visible. (Example: Refer `weather-large.json` in Visualizer)

Reply:
Resolved